<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Cooper Lab - Courses - BINF 6205/8205</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/business-frontpage2.css" rel="stylesheet">

  </head>

  <body>

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
      <div class="container">
        <a class="navbar-brand" href="index.html">Back to Cooper Lab Home</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item active">
              <a class="nav-link" href="#info">Course Description</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="#book">Reading</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="#grading">Grading</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="#topics">Course Topics</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <!-- Header with Background Image -->
    <header class="business-header">
      <div class="container">
        <div class="row">
          <div class="col-lg-12">
		  <h1 class="display-3 text-center text-black mt-4"><strong>BINF 6205/8205: Computational Molecular Evolution</strong></h1>
          </div>
        </div>
      </div>
    </header>

    <!-- Page Content -->
    <div class="container">

      <div class="row" id="info">
        <div class="col-sm-8">
	   <h2 class="mt-4">Course Description</h2>
	   <p>This course will cover both the theoretical and practical aspects of performing phylogenetic analyses, with particular emphasis on understanding the differences and applications of the most commonly used methods. Topics to be covered will include: finding the appropriate data and getting it into the right format; using distance-based and probability-based methods to construct phylogenetic trees; performing ancestral state reconstruction to track the evolution of characters; understanding and calibrating the molecular clock to calculate divergence times; incorporating big data into phylogenetics, and estimating population parameters such as population size, growth, and migration rates.</p>
	   <p><strong>Recommended Pre/Co-requisites:</strong> BINF 6201: Molecular Sequence Analysis; BINF 6200: Statistics for Bioinformatics</p>
	   <p><strong>Learning Objectives:</strong></p>
	   <ol>
		<li>Understand the basic concept of a phylogeny: what is represents and how to interpret it.</li>
		<li>Know where to find and how to generate data sets that are appropriate for phylogenetic analyses, and understand how to construct a data set for answering a particular evolutionary question.</li>
      		<li>Gain a solid understanding of the algorithms underlying different methods and know the strengths and weaknesses of each.</li>
      		<li>Be able to use several of the most commonly used phylogenetic software packages, including BEAST, RAxML, MrBayes, as well as several R packages.</li>
      		<li>Know how to interpret trees and how to use them to test various evolutionary models.</li>
	   </ol>
	</div>
        
	<div class="col-sm-4">
          <h2 class="mt-4">Instructor</h2>
          <address>
            <strong>Liz Cooper, Ph.D | Asst. Professor</strong>
            <br>UNC Charlotte | Dept. of Bioinformatics and Genomics
	    <br>Office: Bioinformatics 271
          </address>
          <address>
            <abbr title="Phone">P:</abbr>
            (704) 687-2402
            <br>
            <abbr title="Email">E:</abbr>
            <a href="mailto:#">lizcooper@uncc.edu</a>
          </address>
        </div>
      </div>
      <!-- /.row -->

      <div class="row align-items-center" id="book">
	<div class="col-sm-4 text-center">
		<img class="img-responsive" src="img/phyloHandbook.jpg" alt="">
	</div>
        <div class="col-sm-8">
           <h2 class="mt-4">Optional Reading</h2>
              <p>There is NO required textbook for this course, and any required reading (in the form of journal articles or other papers) will be provided to students via Canvas.  However, there is a book which some students may find helpful, particularly those interested in learning more of the mathematical details underlying genetic distance and maximum likelihood methods: <u>The Phylogenetic Handbook: A practical approach to phylogenetic analysis and hypothesis testing</u>, Edited by Phillippe Lemey, Marco Salemi and Anne-Mieke Vandamme, 2<sup>nd</sup> Edition, Cambridge University Press 2009.</p>
        </div>
      </div>
      <!-- /.row -->

      <div class="row" id="grading">
	<h2 class="mt-4">Homework and Grading</h2>
	    <p>Each class period will consist of a combination of lecture and in-class computer exercises. Each week, students will be expected to complete all parts of the assigned exercises; anything not finished in class must be completed at home and submitted by <strong>6:00PM </strong>on the due date. Students are permitted to work together on the exercises, but must turn in completed assignments individually. Some weeks may also include assigned reading to supplement the lecture material. The final project will require students to analyze a real data set and submit a written report.</p>
	</div>
	<!-- /.row -->
	<div class="row">
	   <div class="col-sm-6">
		<table class="table table-sm">
		   <tr>
			<td data-title="category" data-breakpoints="xs">Class/Homework Exercises</td>
			<td data-title="percent" data-breakpoints="xs">30%</td>
		   </tr>
		   <tr>
			<td>Quizzes</td>
			<td>20%</td>
		   </tr>
		   <tr>
		   	<td>Midterm Exam</td>
			<td>20%</td>
		   </tr>
		   <tr>
			<td>Final Project</td>
			<td>25%</td>
		   </tr>
		   <tr>
			<td>Attendance</td>
			<td>5%</td>
		   </tr>
		</table>
	   </div>
	   <div class="col-sm-6">
		<table class="table table-sm">
		   <tr>
			<td data-title="letter" data-breakpoints="xs">A</td>
			<td data-title="range" data-breakpoints="xs">90-100%</td>
		   </tr>
		   <tr>
			<td>B</td>
			<td>80-89%</td>
		   </tr>
		   <tr>
			<td>C</td>
			<td>70-79%</td>
		   </tr>
		   <tr>
			<td>D</td>
			<td>60-69%</td>
		   </tr>
		   <tr>
			<td>F</td>
			<td>below 60%</td>
		   </tr>
		</table>
	   </div>
	</div>
	<!-- ./row -->

	<div class="row" id="topics">
	   <h2 class="mt-4">List of Course Topics</h2>
	   <p>The course schedule is organized into roughly 4 sections.  In the first part of the course, we will review and discuss how we select certain sequences and process them in order to use them in a phylogenetic analysis.  Next, we discuss different methods for constructing phylogenetic trees with small (single gene) data sets, along with how to compare and evaluate these methods.  After the midterm, we start to look at newer methods we can use to incorporate larger, multi-gene and next-gen sequencing data into a phylogeny. Finally, we look at how we can use phylogenies and phylogenetic inference to look beyond species relationships and answer other evolutionary questions.</p>
	</div>
	<!-- /.row -->
	<div class="row">
	   <div class="table-responsive">
		<table class="table">
		   <thead>
			<tr class="table-success">
				<th scope="col">Section 1</th>
				<th scope="col">Sequence Data for Phylogenetics</th>
			</tr>
		    </thead>
		    <tbody>
			<tr>
				<th scope="row">1.1</th>
				<td>Introduction and History of Phylogenetics</td>
			</tr>
			<tr>
				<th scope="row">1.2</th>
				<td>Multiple Sequence Alignment Methods</td>
			</tr>
			<tr>
				<th scope="row">1.3</th>
				<td>Identifying Homologous Genes</td>
			</tr>
			<tr>
				<th scope="row">1.4</th>
				<td>Gene Family Evolution: Gene Duplication and Gene Loss</td>
			</tr>
		   </tbody>
		</table>
		<table class="table">
		   <thead>
			<tr class="table-primary">
				<th scope="col">Section 2</th>
				<th scope="col">Constructing Trees with Single Genes</th>
			</tr>
		   </thead>
		   <tbody>
			<tr>
				<th scope="row">2.1</th>
				<td>Genetic Distance Methods: UPGMA and Neighbor-Joining</td>
			</tr>
			<tr>
				<th scope="row">2.2</th>
				<td>Maximum Parsimony</td>
			</tr>
			<tr>
				<th scope="row">2.3</th>
				<td>Bootstrapping to Find Branch Support</td>
			</tr>
			<tr>
				<th scope="row">2.4</th>
				<td>Maximum Likelihood Methods (RAxML)</td>
			</tr>
			<tr>
				<th scope="row">2.5</th>
				<td>Combining, Summarizing, and Comparing Trees</td>
			</tr>
			<tr>
				<th scope="row">2.6</th>
				<td>Ancestral State Reconstruction</td>
			</tr>
		    </tbody>
		</table>
		<table class="table">
		    <thead>
			<tr class="table-danger">
				<th scope="col">Section 3</th>
				<th scope="col">Advanced Methods for Larger Datasets</th>
			</tr>
		    </thead>
		    <tbody>
			<tr>
				<th scope="row">3.1</th>
				<td>Bayesian Inference Methods</td>
			</tr>
			<tr>
				<th scope="row">3.2</th>
				<td>Concatenating and Partitioning Multiple Genes</td>
			</tr>
			<tr>
				<th scope="row">3.3</th>
				<td>Coalescent Theory and the Multispecies Coalescent</td>
			</tr>
			<tr>
				<th scope="row">3.4</th>
				<td>Methods for SNP data: Quartets and SNAPP</td>
			</tr>
		     </tbody>
		</table>
		<table class="table">
		     <thead>
			<tr class="table-warning">
				<th scope="col">Section 4</th>
				<th scope="col">Using Trees to Test Evolutionary Models</th>
			</tr>
		      </thead>
		      <tbody>
			<tr>
				<th scope="row">4.1</th>
				<td>Estimating Divergence Times with the Molecular Clock</td>
			</tr>
			<tr>
				<th scope="row">4.2</th>
				<td>Gene Flow and Phylogenetic Networks</td>
			</tr>
			<tr>
				<th scope="row">4.3</th>
				<td>Population Size Changes</td>
			</tr>
		     </tbody>
		</table>
	</div>
	<!-- /.table-responsive -->
	</div>
	<!-- /.row -->

  </div>
  <!-- /.container -->

    <!-- Footer -->
    <footer class="py-5 bg-dark">
      <div class="container">
        <p class="m-0 text-center text-white">Copyright &copy; Liz Cooper 2018</p>
      </div>
      <!-- /.container -->
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="vendor/jquery/jquery.min.js"></script>
    <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  </body>

</html>
